### DISCUSSION TITLE
osu!lazer argon “pro” skin recommendation.

### DISCUSSION THREAD

**Discussion by Ben6889** [ppy/osu#22328] [Ideas]
Whilst testing the argon pro skin, alot of it is great! However for it to earn the “pro” title, i feel a small change can be implemented that would help greatly with reading (mainly bursts, and HD). Essentially, the hit circle on the skin is very thick around the borders which may block off the view of circles beneath it. I have personally found it quite hard to read bursts/stacked notes especially with hd mod enabled, and I'm sure some players would agree when comparing it to actual “pro” skins. I understand it is still in development, but this change would bring a small but important change to the skin.

**Comment by Ben6889** [comment]
I was going to attach video evidence/support, but files are too big. Not worth compressing, already low quality :p

**Comment by Speykious** [comment]
The problem with the current Argon skin is that it is completely opaque. I don't think it's due to a thick border, as it's probably meant to be dimmed eventually (I could be wrong). Ideally, skin transparency becomes just another user-specific setting.

**Reply by Ben6889** [reply]
(mainly for dt plays especially) I hope they do make the hit circles more transparent. Especially since most pro players tend to stick with DT and HDDT. I am happy to wait and see what future adjustments they make to the skin.

**Comment by razovi** [comment]
I support adding transparency to the argon skins since the following examples don't look right to me.

![overlap1](https://user-images.githubusercontent.com/51028359/216557346-98781e8d-8cb1-4eee-b15b-f60299137cd0.png)
![overlap2](https://user-images.githubusercontent.com/51028359/216557411-514b48ec-dd35-464b-b90a-7f8ef32e3b8e.png)

**Reply by Ben6889** [reply]
Yeah i see what you mean on those examples, behind the sliders the hit circles aren’t visible and can create a disadvantage in a pro setting. Slight tweaks such as these can really make a big difference when it comes to playability.

**Reply by eff3ry** [reply]
I think adding transparency to the argon pro skin is a good idea as well and was also looking into it (then finding out the skin is Hard-Coded), The darker background fill needs the transparency, maybe like 65% opacity idk

Would help tremendously when reading how many circles are in a tight burst
